prometheus/prometheus.yml

```diff
@@ -0,0 +1,9 @@
+scrape_configs:
+  # - job_name: "prometheus"
```

Comment:
why is the code commented?

Reply:
Sorry I pushed the code that I was testing. Would change it and push it again.